src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        int randomInt = new Random().nextInt(101);
+        return new Ball(colorSupplier.getRandomColor(), randomInt);
```

Comment:
same lets make Random and ColorSupplier as class level variables, not to create new instances every time we call getRandomBall() method